solver:
  rollouts: 20
  lambda: 0.0001 #0.001
  h: 10.0
  step_size: 0.015
  horizon: 1.0

  cost_ratio: 0.2
  caching_factor: 0.3
  substeps: 1

  gradient_step_size: 1.0
  momentum_step_size: 0.0

  adaptive_sampling: false
  input_variance: [ 0.05, 0.05, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25]

  discount_factor: 1.0
  verbose: false
  threads: 8
  debug_print: false

  use_tree_search: false
  pruning_threshold: 0.05

  logging: true

  filtering: true
  filters_order: [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]
  filters_window: [10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10]

  spline_degree: 3
  spline_dt: 0.1
  spline_step_size: 0.1

  bound_input: true
  u_min: [-0.2, -0.2, -2.0, -1.0, -1.0, -1.0, -1.0, -1.0, -1.0, -1.0, -1.0, -1.0]
  u_max: [ 0.2,  0.2,  2.0,  1.0,  1.0,  1.0,  1.0,  1.0,  1.0,  1.0,  1.0,  1.0]

  log_data: true


initial_configuration: [0.0, 0.0, 0.2, 0.0, -0.52, 0.0, -1.785, 0.0, 1.10, 0.69, 0.04, 0.04]
linear_weight: 100.0
angular_weight: 10.0
obstacle_radius: 0.15
joint_limits: false
holonomic: false
gaussian_policy: true

policy_update_rate: 0.0  # 0.0 execute as fast as possible
reference_update_rate: 10.0
publish_ros: true
ros_publish_rate: 10.0

sequential: false

apply_safety_filter: false

static_optimization: false
sim_dt: 0.01
max_sim_time: 0.0
